### PR TITLE
Add wslink package

### PR DESCRIPTION
**Pull request by banesullivan**
Resolves https://github.com/Kitware/wslink/issues/121

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by banesullivan** [comment]
@jourdain, would you please consent to be listed as a maintainer of this recipe?

**Comment by jourdain** [comment]
I guess...

**Comment by jourdain** [comment]
From what a recipe looks like, it seems fine to me. (I was not sure how complex it was.)

**Comment by banesullivan** [comment]
> From what a recipe looks like, it seems fine to me. (I was not sure how complex it was.)

Not complex at all - everything here is boilerplate and autogenerated using [grayskull](https://github.com/conda-incubator/grayskull)

Once the feedstock lands, it should maintain itself only needing changes if the dependencies change

**Comment by banesullivan** [comment]
@conda-forge/staged-recipes, ready for review and @conda-forge-admin, please ping team

File: recipes/wslink/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  home: https://github.com/kitware/wslink
+  summary: Python/JavaScript library for communicating over WebSocket
+  license: BSD-3-Clause
+  license_file: src/wslink/LICENSE
+
+extra:
+  recipe-maintainers:
+    - banesullivan
+    - jourdain
+    - kwrobot
```
Comment (ocefpaf) [inline recipes/wslink/meta.yaml:43]: @kwrobot do agree to be a maintainer here.

Comment (dopplershift) [reply recipes/wslink/meta.yaml:43]: I just looked, that looks like a Kitware robot account.

Comment (banesullivan) [reply recipes/wslink/meta.yaml:43]: Indeed, this is an account we use to ensure Kitware as an organization has access to our repositories, etc. For precedence, we've added this account on other feedstocks

Comment (banesullivan) [reply recipes/wslink/meta.yaml:43]: FYI https://github.com/conda-forge/staged-recipes/pull/18785#discussion_r860123347

**Pull request merged**